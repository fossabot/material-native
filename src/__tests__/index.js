'use strict';

test('main script has expected exports', () => {
	expect(Object.keys(require('../')).sort()).toMatchSnapshot();
});
