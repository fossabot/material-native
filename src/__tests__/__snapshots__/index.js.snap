// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main script has expected exports 1`] = `
Array [
  "AlertDialog",
  "Avatar",
  "BorderlessRipple",
  "Button",
  "Checkbox",
  "CircleHighlight",
  "CircleRipple",
  "CommunityIcon",
  "ConfirmDialog",
  "CoreCheckbox",
  "CoreDialog",
  "CoreRadio",
  "CoreRipple",
  "Dialog",
  "DialogModal",
  "Divider",
  "FAB",
  "Icon",
  "ItemRipple",
  "LabeledCheckbox",
  "LabeledRadio",
  "MaterialText",
  "MaterialTheme",
  "PickerDialog",
  "Radio",
  "RectRipple",
  "Subheader",
  "TextField",
  "ThemeProvider",
  "colors",
  "contrast",
  "createIconComponent",
  "easing",
  "elevation",
  "getMaterialTheme",
  "shades",
  "typo",
  "withMaterialStyles",
  "withMaterialTheme",
]
`;
